taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -0,0 +1,93 @@
+import { Paper, Stack, TextField, Typography } from "@mui/material";
+import { useEffect, useState } from "react";
+
+const ListAllOtherProperties = ({ props }) => {
+    const [nodeObject, setNodeObject] = useState(null);
+    useEffect(() => {
+        setNodeObject(props);
+    }, [props])
+
+    let lc = '';
+    let toBeRendered = {}
+    let languageNames = new Intl.DisplayNames(['en'], {type: 'language'});
+
+    if (nodeObject) {
```

Comment:
I may not be getting the whole idea , but, maybe pass this to the useEffect ? and set the state of line 5 to what you will use at the end?

Reply:
Since setNodeObject is async, it isn't possible to traverse through it's keys inside useEffect. So, I'm checking it separately after the useEffect.

Also, I'm maintaining two separate objects (`nodeObject` and `toBeRendered`) so that after the user has done all the edits, I'll be able to update nodeObject using values from `toBeRendered` and give a POST request of nodeObject to the backend. Does this make sense?

Reply:
yeah makes sense, thanks